packages/@aws-cdk/pipelines/README.md

```diff
@@ -867,11 +882,11 @@ because those values are passed in directly to the underlying `codepipeline.Pipe
 
 Docker can be used in 3 different places in the pipeline:
 
-* If you are using Docker image assets in your application stages: Docker will
+- If you are using Docker image assets in your application stages: Docker will
```

Comment:
Why this change?

Reply:
This came automatically with markdownlint.